packages/design-system/src/molecules/tokenCard/tokenCard.test.tsx

```diff
@@ -0,0 +1,82 @@
+import { fireEvent, render, screen } from '@testing-library/react-native'
+import { TryrollTestProvider } from '../../providers'
+import { CHAIN_ID_MAIN_NET } from '../../web3'
+import * as utils from '../../utils'
+import { TokenCard } from '.'
+
+const TOKEN_ADDRESS = '0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1'
+const TOKEN_NAME = 'Wagmi'
+const TOKEN_SYMBOL = 'WAGMI'
+
+jest.mock('../../utils', () => ({
+  ...jest.requireActual('../../utils'),
+  openLink: jest.fn(),
+}))
+
+const copy = jest.fn()
+jest.mock('../../hooks', () => ({
+  ...jest.requireActual('../../hooks'),
+  useCopyOnClipboard: () => copy,
+}))
+
+describe('TokenCard', () => {
+  beforeEach(() => {
+    render(
+      <TokenCard
+        logo={TOKEN_ADDRESS}
+        address="0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1"
+        name={TOKEN_NAME}
+        symbol={TOKEN_SYMBOL}
+        chainId={CHAIN_ID_MAIN_NET}
+      />,
+      {
+        wrapper: TryrollTestProvider,
+      },
+    )
+  })
+  it('Renders short address', async () => {
```

Comment:
Can you use lowercase letters for `it`, please? We have to be consistent within the codebase